### ISSUE TITLE
missing spaces around "=" operator

### ISSUE THREAD

**Issue by braoult** [juampapo546/Fan-control#5]
lines 72 and 109 will always eval as "true" :
```
if [ $fan2_manual="0" ]; then
```
and
```
if [ $fan3_manual="0" ]; then
```
should be rewritten as :
```
if [ "$fan2_manual" = "0" ]; then
```
and
```
if [ "$fan3_manual" = "0" ]; then
```

**Comment by JuampiCarosi** [comment]
Oh cool, still learning about bash scripting 😅
I'll merge changes today, thanks for the tip!

**Comment by JuampiCarosi** [comment]
I see you've been doing some changes, I'll be happy to merge PRs 😀

**Comment by braoult** [comment]
I will be happy to do so, but I did not finish the tests...
Will you be happy with the script being bash-only (shebang line) ?

**Comment by JuampiCarosi** [comment]
Yeah, np. Back then I did it like this bc it was the way I found how to do it with some quick googling :)

**Comment by braoult** [comment]
here it is: #6

**Comment by JuampiCarosi** [comment]
Merged #6

**Comment by braoult** [comment]
fixed with #6
